docs/source/gateway/index.rst

```diff
@@ -140,7 +140,7 @@ Here's an example of how to send a chat request using the :ref:`gateway_fluent_a
 
     response = query(
         "chat",
-        {"messages": [{"role": "user", "content": "What is the best day of the week?"}]},
```

Comment:
trailing commas look a bit strange in our docs

Reply:
We format example code blocks using black. Black adds trailing commas.

Reply:
IMO, trailing commas are fine.

Reply:
The reason we format code blocks is prevent silly syntax errors.

Reply:
The lint check would fail without trailing commas.

Reply:
Personal preference, but I don't love seeing them in docs. I don't feel strongly

Reply:
Got it, then let's restore trailing commas.